src/arr/compiler/compile-lib.arr

```diff
@@ -419,10 +424,26 @@ fun compile-module(locator :: Locator, provide-map :: SD.StringDict<CS.Provides>
               end
             desugared := nothing
             add-phase("Type Checked", type-checked)
-            cases(CS.CompileResult) type-checked block:
+            var stepified =
```

Comment:
Why is stepified done inside type checking phase?

Reply:
Good question. I was copying the approach I used before, which put it at this stage, but I don't recall why, and for all I know the reason why may no longer apply. So I'll move it to happen right after `desugar`. ("It's just a cached thought," I say, as I tear down Chesterson's fence.)